app/graphql/mutations/create_card.rb

```diff
@@ -0,0 +1,18 @@
+module Mutations
+  class CreateCard < Mutations::BaseMutation
+    argument :text, String, required: true
+    argument :board_id, ID, required: true
+    argument :column_id, ID, required: true
```

Comment:
Please, sort fields alphabetically.
```suggestion
    argument :board_id, ID, required: true
    argument :column_id, ID, required: true
    argument :text, String, required: true
```

Reply:
👍